url-updater/src/main/java/com/devonfw/tools/ide/url/folderhandling/jsonfile/JsonDataBlockForSpecificUrl.java

```diff
@@ -0,0 +1,26 @@
+package com.devonfw.tools.ide.url.folderhandling.jsonfile;
+
+import java.util.Set;
+
+public class JsonDataBlockForSpecificUrl {
+
+	private Set<Double> urlHashes;
```

Comment:
why should we use `Double` for the hashes?
I would expect Integer or Long as a hash is not a decimal number.